docs/contracts.rst

```diff
@@ -834,9 +833,9 @@ Arguments for Base Constructors
 ===============================
 
 Derived contracts need to provide all arguments needed for
-the base constructors. This can be done at two places::
+the base constructors. This can be done in two ways::
```

Comment:
Probably we could split the following example into two to clearly explain the two ways.